### PR TITLE
Fix bugs introduced by v2 migration

### PR DESCRIPTION
**Pull request by lidongze91**

### CODE REVIEW

**Comment by lidongze91** [comment]
V2 migration introduced bugs such that app on v1 would crash with no setup for channel and audiomanager.

File: audioplayer/android/src/main/java/bz/rxla/audioplayer/AudioplayerPlugin.java [hunk 1]
```diff
@@ -46,8 +38,14 @@ public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBindin
   // depending on the user's project. onAttachedToEngine or registerWith must both be defined
   // in the same class.
   public static void registerWith(Registrar registrar) {
-    final MethodChannel channel = new MethodChannel(registrar.messenger(), "audioplayer");
-    channel.setMethodCallHandler(new AudioplayerPlugin());
+    AudioplayerPlugin instance = new AudioplayerPlugin();
+    instance.initInstance(registrar.messenger(), registrar.context().getApplicationContext());
```
Comment (mehmetf) [inline audioplayer/android/src/main/java/bz/rxla/audioplayer/AudioplayerPlugin.java:42]: I think you can just pass `registrar.context()` here.

File: example/lib/main.dart [hunk 2]
```diff
@@ -157,7 +157,7 @@ class _AudioAppState extends State<AudioApp> {
           children: [
             Text(
               'Flutter Audioplayer',
-              style: textTheme.headline6,
+              style: textTheme.headline,
```
Comment (mehmetf) [inline example/lib/main.dart:160]: Why this change? it does not compile?

Comment (lidongze91) [reply example/lib/main.dart:160]: Yeah. this is a typo.

File: audioplayer/android/src/main/java/bz/rxla/audioplayer/AudioplayerPlugin.java (hunk 1 above)
Comment (lidongze91) [reply audioplayer/android/src/main/java/bz/rxla/audioplayer/AudioplayerPlugin.java]: Done
